AUTHORS

```diff
@@ -0,0 +1,17 @@
+# This is the list of OpenROAD's significant contributors.
+#
+# This does not necessarily list everyone who has contributed code,
+# especially since many employees of one corporation may be contributing.
+# To see the full list of contributors, see the revision history in
+# source control.
+
+Andrew Kennings
+Google LLC
+Iowa State University
+Nefelus, Inc.
+Parallax Software, Inc.
+Peter Gadfort
```

Comment:
@gadfort let me know if you prefer "Zero ASIC Corporation"

Reply:
Sorry got sidetracked, I think this is fine for now

Reply:
You can always PR a change if desired